### ISSUE TITLE
Windows MSIX issue: python 3.11beta in msix cant be installed on windows 10 1809 arm64

### ISSUE THREAD

**Issue by busylog** [python/cpython#95733]
# Bug report

I cant install python 3.11 beta on windows 10 1809/17763 arm64 via msix.

I got the msix file via https://store.rg-adguard.net/ , productid is 9NRWMJP3717K

related command and log:
```
 Add-AppPackage -path .\PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.Msix

 Get-AppPackageLog -ActivityID ff7676ee-9614-0006-8b09-8bff1496d801

Time                      ID           Message
----                      --           -------
8/6/2022 10:18:53 AM      603          Started deployment Add operation on a package with main parameter
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.Msix and
                                       Options 0 and 0. See http://go.microsoft.com/fwlink/?LinkId=235160 for help
                                       diagnosing app deployment issues.
8/6/2022 10:18:54 AM      157          The app package signature was validated for core content of the app package
                                       published by CN=4975D53F-AA7E-49A5-8B49-EA4FDC1BB66B. Payload won't be
                                       validated until the files are read.
8/6/2022 10:18:54 AM      135          error 0xC00CE014: App manifest validation error: The app manifest must be valid
                                       as per schema: Line 9, Column 6, Note: The schema for MaxVersionTested
                                       specified does not recognize XML fields with namespace
                                       "http://schemas.microsoft.com/appx/manifest/desktop/windows10/6". Please ensure
                                       that you have the correct MaxVersionTested specified. Reason: Element '{http://s
                                       chemas.microsoft.com/appx/manifest/desktop/windows10/6}RegistryWriteVirtualizati
                                       on' is unexpected according to content model of parent element
                                       '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}Properties'.

8/6/2022 10:18:54 AM      465          error 0x80080204: Opening the package from location
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.Msix
                                       failed.
8/6/2022 10:18:54 AM      403          error 0x80080204: Failure to get staging session for: file:///C:/Users/USER/Downloa
                                       ds/PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.Msix.
8/6/2022 10:18:54 AM      404          AppX Deployment operation failed for package  with error 0x80080204. The
                                       specific error text for this failure is: error 0xC00CE014: App manifest
                                       validation error: The app manifest must be valid as per schema: Line 9, Column
                                       6, Note: The schema for MaxVersionTested specified does not recognize XML
                                       fields with namespace
                                       "http://schemas.microsoft.com/appx/manifest/desktop/windows10/6". Please ensure
                                       that you have the correct MaxVersionTested specified. Reason: Element '{http://s
                                       chemas.microsoft.com/appx/manifest/desktop/windows10/6}RegistryWriteVirtualizati
                                       on' is unexpected according to content model of parent element
                                       '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}Properties'.
```

# Your environment

- CPython versions tested on: 3.11
- Operating system and architecture: windows 10 17763 arm64

note: python 3.11 is the first version support arm64 windows. so I cant test previous x64 version.

maybe related: https://github.com/python/cpython/commit/3a5afc14e16370c1f4f72d43cb553298ad9a1fa4

**Comment by busylog** [comment]
https://github.com/python/cpython/issues/90796#issuecomment-1093944049 admits it does not work on 1809.
it is a low priority bug since very few person knows how to do keep windows update on 1809. msix not working isnt a big issue, I can install exe/msi version after final release.

**Comment by busylog** [comment]
maybe I have found the solution!
https://github.com/PowerShell/PowerShell/blob/master/assets/AppxManifest.xml
```
<Package IgnorableNamespaces="uap mp rescap desktop6"
```
cc @zooba

**Comment by busylog** [comment]
```
Add-AppxPackage -Register C:\temp\PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0\AppxManifest.xml
Get-AppPackageLog -ActivityID ec57611a-ab76-0005-b175-57ec76abd801

Time                      ID           Message
----                      --           -------
8/9/2022 6:49:42 AM       603          Started deployment Register operation on a package with main parameter
                                       AppxManifest.xml and Options DevelopmentModeOption and 0. See
                                       http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app
                                       deployment issues.
8/9/2022 6:49:42 AM       216          App manifest validation warning: Declared namespace
                                       http://schemas.microsoft.com/appx/manifest/desktop/windows10/6 is inapplicable,
                                       it will be ignored during manifest processing.
8/9/2022 6:49:42 AM       10002        Creating Resiliency File C:\ProgramData\Microsoft\Windows\AppRepository\------.rsl
                                       c for Register Operation on Package
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.
8/9/2022 6:49:42 AM       607          Deployment Register operation on package
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0 has been
                                       de-queued and is running for user SID
                                       ------.
8/9/2022 6:49:42 AM       854          Successfully added the following uri(s) to be processed:  C:\temp\PythonSoftware
                                       Foundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0\AppxManifest.xml.
8/9/2022 6:49:42 AM       855          Finished resolving action lists. DeploymentRequest action lists:
                                       addPackageList:
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0.
8/9/2022 6:49:42 AM       440          Package PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0
                                       with Windows Store origin cannot be sideloaded or installed using developer
                                       mode option.
8/9/2022 6:49:42 AM       605          The last successful state reached was Resolved. Failure occurred before
                                       reaching the next state Approved. hr: 0x80073CFF
8/9/2022 6:49:42 AM       401          Deployment Register operation with target volume C: on Package
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0 from:
                                       (AppxManifest.xml)  failed with error 0x80073CFF. See
                                       http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app
                                       deployment issues.
8/9/2022 6:49:42 AM       404          AppX Deployment operation failed for package
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0 with error
                                       0x80073CFF. The specific error text for this failure is: Package
                                       PythonSoftwareFoundation.Python.3.11_3.11.181.0_arm64__qbz5n2kfra8p0 with
                                       Windows Store origin cannot be sideloaded or installed using developer mode
                                       option.
```

note this is after IgnorableNamespaces mod.

**Comment by busylog** [comment]
so my impression is, 1809 users will be thrown out if windows10/6(desktop6) is used and not adding IgnorableNamespaces. it does different on isolation capabilities
ref: https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/what-s-changed-in-windows-10

**Comment by busylog** [comment]
installed successfully after deleting APPXMETADATA, APPXBLOCKMAP, APPXSIGNATURE, [CONTENT_TYPES] to resolve 0x80073CFF error.

**Comment by zooba** [comment]
> installed successfully after deleting APPXMETADATA, APPXBLOCKMAP, APPXSIGNATURE, [CONTENT_TYPES] to resolve 0x80073CFF error.

Nice! So adding the field should handle this issue. I can't guarantee it'll get into 3.11 (we already dropped support for 1809, because Microsoft has, and we're already in RC), but shouldn't be any harm in adding it to the next release. Might even get back into 3.11.1 if we test a 3.12 release in the meantime.

**Comment by zooba** [comment]
Oh actually, could you check that it _fails_ to install under the same circumstances? Often the dev install is more lenient than the regular case, so be good to check that it fails without the deleted metadata and without the IgnorableNamespaces attribute.

**Comment by busylog** [comment]
it does fail on the same error complaining windows10/6 on dev mode

**Comment by zooba** [comment]
This will be fixed in 3.12. I don't think we want this change in 3.11 right now, though it seems likely to be safe enough (but our bar for changes after RC is very high). Might be able to get it into 3.11.1
